src/logic/operate.js

```diff
@@ -0,0 +1,25 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result;
+  const a = Big(numberOne);
+  const b = Big(numberTwo);
+
+  if (operation === '+') {
+    result = a.plus(b);
+  } else if (operation === '-') {
+    result = a.minus(b);
+  } else if (operation === 'x') {
+    result = a.times(b);
+  } else if (operation === '÷') {
+    if (numberTwo !== '0') {
+      result = a.div(b);
+    } else {
+      return 'Error';
+    }
+  } else if (operation === '%') {
+    result = a.times(0.01);
```

Comment:
- [ ] Instead of using `times` here, try using the `modifier` function to turn a number into a percentage 😉